test/requirements.txt

```diff
@@ -1,5 +1,4 @@
-apache-airflow==1.10.11
-apache-beam==2.23.0
-six==1.14.0
-dill==0.3.0
+apache-airflow>=1.10
```

Comment:
In this case, we allow installing 2.0 airflow which will be probably incompatible

Reply:
It was done on purpose - validate that bigflow-generated DAGs are compatible with latest stable version of 'airflow' (this dependency used only for development/ci). As we actually don't have any control on what version of Airflow is used by our clients.